openml/datasets/functions.py

```diff
@@ -815,12 +815,16 @@ def edit_dataset(
 ) -> int:
     """
       Edits an OpenMLDataset.
-      Specify atleast one field to edit, apart from data_id
-       - For certain fields, a new dataset version is created : attributes, data,
-       default_target_attribute, ignore_attribute, row_id_attribute.
+      Specify atleast one field to edit, apart from data_id.
```

Comment:
`at least` requires a space

Reply:
After some consideration, I would word this slightly differently (because this might imply you are editing the dataset id too):

> In addition to providing the dataset id of the dataset to edit (through `dataset_id`), you must specify a value for at least one of the optional function arguments, i.e. one value for a field to edit.

It's a little bit more verbose, but I think it is a bit clearer, especially since this is end-user facing documentation (i.e. this function should not only be called by us internally, but also users that only use the package and not develop it).